### PR TITLE
Feature/3 add folders for organizing script container file etc

### LINKED ISSUE #3: Add folders for organizing scripts, container files etc
**What would you like to be added**:
Althrough the package build is still in a PoC state, we should start to add some folders to this repo and reorganize some of the files so it's easier for external to understand what this package build consists of. 

Currently, all the scripts are added to the root directory, making it hard to follow and understand the package build.

**Why is this needed**:
This change is needed to make it easier for newcomers and external to understand what the package build conists of.

### PR DESCRIPTION
**Pull request by alee-cd**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #3 

**Special notes for your reviewer**:

Moved files into bin/, scripts/, conf, misc/, container/ folders and created a simple README.md file.

### CODE REVIEW

**Comment by ghost** [comment]
@alee-ntap Thank you for your contribution.

**Review by Vincinator — APPROVED** [review]
Looks good. 

Maybe we need to rename the `scripts` in the future. `scripts` does not intuitively tell me that it is used for github actions only. 

But works for me for now

**Comment by alee-cd** [comment]
> Maybe we need to rename the `scripts` in the future. `scripts` does not intuitively tell me that it is used for github actions only.

Sure. Any suggestion for the name?

**Comment by Vincinator** [comment]
Hm, maybe it would help if we could move the scripts folder to `.github/scripts/`, then it would be clear that it is for github actions.

**Comment by alee-cd** [comment]
After one more review and discuss with @Vincinator . It makes more sense to move everything will be installed into the container into `container/` folder. And then leave the rest on the root path eg: the `misc/mini_sudo.c` and `scripts/gh_release`. I will update these into this PR.
